src/V3SignalStrength.h

```diff
@@ -0,0 +1,32 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Handle signal strength
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2003-2022 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#ifndef VERILATOR_V3SIGNAL_STRENGTH_H_
+#define VERILATOR_V3SIGNAL_STRENGTH_H_
+
+#include "config_build.h"
+#include "verilatedos.h"
+
+class AstNetlist;
+
+//============================================================================
+
+class V3SignalStrength final {
+public:
+    static void handleStrength(AstNetlist* nodep);
```

Comment:
```suggestion
    static void strengthAll(AstNetlist* nodep);
```
Consistent with other such methods (`constifyAll`, `assertAll`, etc)

Reply:
Done